HW1/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>   
+	<head>
+		<title> CICI Mug</title>
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="http://s3.amazonaws.com/codecademy-content/courses/ltp/css/bootstrap.css">
+		<link rel ="stylesheet" href="css/style.css">
+	</head>
+
+	<body>
+		<div class = 'container'>
+			<header>
+				<h1 class='largetitle'> Orange Udacity Mug </h1>
+			</header>
+
+			<div class="row">
+
+				<div class="col-sm-5">
+					<div class='leftCol'>
+						<img src="mug.png" class="img-responsive">
+			            <section>
+			                <header>
+			                    <h2>Main Features</h2>
+			                </header>
+			                <ul>
+			                    <li>Holds your favorite drink</li>
+			                    <li>Energizing color</li>
+			                    <li>Reminds you to learn</li>
+			                </ul>
+			            </section>		
+					</div>
+	            </div>
+
+		        <div class="col-sm-7">
+		        	<div class='rightCol'>
+		        			<h2>A Good Way to Start a Day</h2>
+		        			<p>
+		        				After two hours struggling with a margin I started thinking if the idea of quitting my job was really a good decision. I told my boss that after several years I wasn't advancing in my career and that from now on I was going to be the owner of my fate. I was going to take a nanodegree! "A nano what?" She managed to say. I didn't bother to answer. I did all the paperwork and left my job. Now I'm stuck here with a magin-left...The style guide says: "Margin left? Don't use that. Forget it". But how do I align this text with the photo above? I didn't have these kinds of problems at the office. Why not continue with my previous life where I had the same problems but with the same standard solutions? I'm not built for this. I'd rather talk with my former boss and tell her I made a terrible mistake. That I had a high fever and was hallucinating... I want the security and boredom that rules in the office.
+		        			</p>
+
+		        			<p>
+		        				I go to the kitchen. Grab this orange mug and start pouring coffee. I stare at the mug with this big "U". Ah, Udacity. I remember this girl tossing an angry bird at her co-worker who was furiously scribbling code on his computer. I remember he said something like "if you are stuck with an issue most probably someone before had the same problem, posted it on Stackoverflow and another someone answered" I go back to my computer and start googling. Twenty minutes later...nothing. I sip some coffe from my Udacity mug and then I think "What if I put the text in the same div as the photo? Bootstrap will take care of the rest..." It's just a hunch. But, wait! It works, they perfectly align! I raise my arms and flavor victory as if I were at the stadium. The public claps and cheers. Yeah, I'm becoming a ninja at coding!
+		        			</p>
+
+		        	</div>
+		        </div>
+
+	        </div>
+        </div>
```

Comment:
please make sure your indention here is perfect, that will increase the readability of your code, and make you a professional programmer